bat-nb/README.md

```diff
@@ -0,0 +1,10 @@
+## bat
```

Comment:
can you please split this PR into having a directory for each separately? i.e. bat/ and nb/ ?